analyzers/nodejs/nodejs.go

```diff
@@ -261,6 +263,19 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	return yarn.FromProject(filepath.Join(a.Module.BuildTarget, "package.json"), filepath.Join(a.Module.BuildTarget, "yarn.lock"))
 }
 
+// fixLegacyBuildTarget ensures that legacy behavior stays intact but users are warned if it is implemented.
+func fixLegacyBuildTarget(target string) string {
+	if strings.HasSuffix(target, "/package.json") {
```

Comment:
Here and below, use `filepath.Separator` instead of hardcoding `/`. This will do the right thing on Windows, which uses `\`.